### PR TITLE
PD-978 Rework TrueNASUpgrades as Software Releases

### PR DESCRIPTION
**Pull request by Mrt134**
Update the content to linearly communicate existing releases, using Nightly or Legacy releases, then schedules for future releases.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/TrueNASUpgrades/_index.md [hunk 1]
```diff
@@ -1,29 +1,97 @@
 ---
-title: "Release Schedules"
+title: "Software Releases"
 description: "Centralized schedules and upgrade charts for software releases."
 weight: 25
 related: false
+aliases:
+ - /scale/gettingstarted/upgrades/
 ---
```
Comment (DjP-iX) [inline content/TrueNASUpgrades/_index.md:9]: I think we need at least an h2 here, if not maybe a sentence or two letting readers know that these tables contain links to download different build versions. The sidebar toc looks a little odd with the first two items being indented h3s also, but adding a header at the top would help with that too

File: content/TrueNASUpgrades/_index.md [hunk 2]
```diff
@@ -1,29 +1,97 @@
...
+Legacy releases can only be used by downloading the .iso file and freshly installing to the hardware.
+See the [Documentation Archive](https://www.truenas.com/docs/archive/) for content related to these releases.
+
+## Upgrade Paths
 
 {{< columns >}}
+**TrueNAS SCALE**
 
-{{< releaselist name=core-releases defaultTab=2 >}}
+{{< mermaid class="mermaid_sizing" >}}
```
Comment (DjP-iX) [inline content/TrueNASUpgrades/_index.md:50]: Text on the mermaid charts is really small now that the sidebar toc has the content area squished down

File: content/TrueNASUpgrades/_index.md [hunk 3]
```diff
@@ -1,29 +1,97 @@
...
+
+Individual releases within a major version are labeled in this section as:
+* **Early** - public release of a unstable version that is either not feature complete or has more testing cycles planned.
+  Follows an ALPHA, BETA, and RC naming convention.
+
+* **Stable** - public release of a feature complete and internal and community tested major version.
+  Follows a .0 naming convention.
+
+* **Maintenance** - public release with software bugfixes and additional internal and community testing.
+  Follows a .# naming convention, with small scope maintenance releases ("hotpatches") following a .#.# convention.
```
Comment (DjP-iX) [inline content/TrueNASUpgrades/_index.md:79]:
```suggestion
  Follows a .# naming convention, with small-scope maintenance releases ("hotpatches") following a .#.# convention.
```

**Review by micjohnson777 — APPROVED** [review]
Looks great!

**Review by DjP-iX — APPROVED** [review]

**Review by DjP-iX — APPROVED** [review]

**Pull request merged**